<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-PLAI3UQ" failures="2" tests="2" name="My Test3" time="0.414" errors="0" timestamp="2024-10-23T00:00:32 IST">
  <testcase classname="testCases.TC001_Tutorialsninja_AccountRegistration_Test" name="Test_Account_Registration" time="0.068">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC001_Tutorialsninja_AccountRegistration_Test.Test_Account_Registration(TC001_Tutorialsninja_AccountRegistration_Test.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- Test_Account_Registration -->
  <testcase classname="testCases.TC002_Tutorialsninja_Login_Test" name="Login_Test" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC002_Tutorialsninja_Login_Test.Login_Test(TC002_Tutorialsninja_Login_Test.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- Login_Test -->
</testsuite> <!-- My Test3 -->
